src/clr-ui/.stylelintrc

```diff
@@ -3,25 +3,21 @@
   "defaultSeverity": "error",
   "syntax": "scss",
   "plugins": [
-    "stylelint-scss",
-    "stylelint-declaration-use-variable",
-    "stylelint-no-px"
+    "stylelint-scss"
   ],
   "ignoreFiles": [
+    "src/clr-angular/utils/_normalize.scss"
   ],
   "rules": {
-    "sh-waqar/declaration-use-variable": [["color", { ignoreValues: ["inherit", "red", "blue"] }]],
-    "meowtec/no-px": [true, { "ignore": ["1px"] }],
-    "color-no-hex": true,
     "no-descending-specificity": null,
     "no-duplicate-selectors": null,
     "at-rule-no-unknown": null,
     "scss/at-rule-no-unknown": true,
-    "max-nesting-depth": 3,
-    "block-closing-brace-newline-after": "always",
```

Comment:
The fix for the `block-closing-brace-newline-after` is on master now so if you rebase you should be able to revert this file change and everything build.